packages/zowe-explorer/i18n/sample/src/command/MvsCommandHandler.i18n.json

```diff
@@ -4,12 +4,12 @@
   "issueMvsCommand.undefined.profilename": "Operation Cancelled",
   "issueMvsCommand.noProfilesLoaded": "No profiles available",
   "issueMvsCommand.checkProfile": "Profile is invalid",
-  "issueMvsCommand.command.hostname": "Select a command to run immediately against ",
+  "issueMvsCommand.command.hostname": "Select a MVS command to run immediately against ",
```

Comment:
I think it should be `an MVS command` since it starts with a vowel sound...sorry for nitpicking XD

https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Abbreviations/faq0005.html